app/src/main/java/com/infomaniak/mail/ui/main/newMessage/AiPromptFragment.kt

```diff
@@ -82,6 +93,20 @@ class AiPromptFragment : Fragment() {
         closeButton.setOnClickListener {
             (parentFragment as NewMessageFragment).closeAiPrompt()
         }
+
+        generateButton.setOnClickListener {
+            generationLoader.isVisible = true
+            generateButton.isInvisible = true
+
+            aiPromptViewModel.generateAiProposition(newMessageViewModel.aiPrompt)
```

Comment:
Is doesn't seem logical that aiPrompt is in newMessageViewModel instead of AiPromptViewModel, is there no possibility of refactor ?

Reply:
The main idea was for values that represent the state of the view to be in the NewMessageViewModel and the rest (apparently only api calls related stuff) in the ai view model. But I don't know if there is a common way of splitting code for this kind of situation. Maybe it would make sens to move aiPrompt and isAiPromptOpened inside the Ai view model to at least free up some space in the new message view model which is long enough as it is

Reply:
Move it all to AI VM.

Reply:
Done